types/Organization.js

```diff
@@ -183,7 +182,6 @@ const OrgDonationLedger = S.array()
       .prop('id', S.string())
       .prop('name', S.string())
       .prop('registry', S.string())
-      .prop('maintainers', S.array().items(MaintainerBase))
```

Comment:
I don't think the ledger should send back the maintainers list, so I removed it. Also the MaintainerBase type was old af so I trashed it.